client/src/Pages/NarratorWaitingScreen.jsx

```diff
@@ -1,17 +1,42 @@
 import React from 'react';
-import {Button} from 'antd';
+import {Button, Row, Col} from 'antd';
 
 export default class NarratorWaitingScreen extends React.Component{
 
     render(){
         return(
-            <div>
-                You are the Narrator...
-                <br/>
-                Press Start to begin the game
-                <br/> 
-                <Button>Start</Button>
-            </div>
+            <>
+            <Row style = {{paddingTop: "3vh", paddingBottom: "3vh"}}>
+                <Col xs = {0} md = {8}/>
+                <Col xs = {24} md = {8} style = {{textAlign: "center", fontFamily: "minecraft", fontSize: "200%"}}>
+                    You are the Narrator
+                </Col>
+                <Col xs = {0} md = {8}/>
```

Comment:
The pattern of "padding column" followed by "content column" followed by "padding column" shows up a lot in this code. I'm not sure I understand it. Wouldn't it look the same if we just had the "content column" but it was always centered? The `xs` and `md` values could stay the same.

If that's the case, I'd much prefer that method. Having triple the number of columns makes the code a lot harder to read.

Reply:
I was able to find an easy solution using justify-content, that provides a great solution. This also made me think about eventually making the grid layout into flex-boxes in the future if we have the time.